packages/calling-component-bindings/src/callControlSelectors.ts

```diff
@@ -43,7 +43,7 @@ export const microphoneButtonSelector: MicrophoneButtonSelector = reselect.creat
   (callExists, isMuted, deviceManager) => {
     const permission = deviceManager.deviceAccess ? deviceManager.deviceAccess.audio : true;
     return {
-      disabled: !callExists || !permission,
+      disabled: !callExists || !permission || !deviceManager.microphones.length,
```

Comment:
does this still allow the speakers to be changes using the microphone dropdown? (callwithchat)

Reply:
Great catch! Updated: 
![image](https://user-images.githubusercontent.com/2684369/175392246-fa6c652d-d16f-42e2-b0f5-10f0b29639cb.png)